rviz_common/src/rviz_common/visualization_manager.cpp

```diff
@@ -225,7 +226,15 @@ VisualizationManager::VisualizationManager(
 
   connect(this, SIGNAL(timeJumped()), this, SLOT(resetTime()));
 
-  executor_->add_node(rviz_ros_node_.lock()->get_raw_node());
+  if (!mgr_) {
+    mgr_ = std::make_shared<ros_integration::ComponentManager>(executor_);
```

Comment:
Should we allow users to configure the name of the component manager and pass it along as an argument here?
There could be name clashes with other component containers running. It would also make adding nodes to this container less ambiguious by being able to specify the right name.